web/src/i18n/locales/en.yaml

```diff
@@ -172,18 +172,18 @@
     description: 
       title: "Activity Overview"
       helpText: "Summarize the needs of the activity for which requested funding will be used (e.g. administrative, outreach, auditing, Public Health Registry Development, etc)"
-      reminder: "This should be very brief."
+      reminder: "The activity overview should be a brief summary of the activity needs."
       summary: 
         subheader: "Brief Activity Summary"
       detail:
         standard:
-          subheader: "Describe the activity and include relevant information."
+          subheader: "The activity summary should be brief, there will be additional opportunities to expand in other sections."
         hie:
           subheader: "Describe the activity in detail including images, diagrams, or other relevant information."
           helpText: "The activity details must also include a description of three items: how the activity is linked to meaningful use; value proposition to providers, and sustainability plan describing how the activity will be sustained after HIE DDI funding is exhausted.	Additionally, include diagrams and descriptions of other HIE linkages as needed if they provide more context for how you'll complete the activity."
       alternatives: 
         subheader: "Statement of alternative considerations and supporting justification"
-        helpText: "If no substantive changes have been made to this activity, include a statement referring to the Statement of Alternative Considerations contained in previous versions of the IAPD or in the SMHP. If substantive changes have been made, include a statement describing the alternatives your state considered when making the change."
+        helpText: "If no substantive changes have been made to this activity, include a statement referring to the Statement of Alternative Considerations contained in previous versions of the IAPD update or in the SMHP. If substantive changes have been made, include a statement describing the alternatives your state considered when making the change."
```

Comment:
For this line: "If substantive changes have been made, include a statement describing the alternatives your state considered when making the change."

Suggestion (feel free to revise - point is that also covers new activities): "If substantive changes have been made OR *this is a new activity*, include a statement describing what alternative considerations were made before selecting the strategy for implementation."